pkg/state/state.go

```diff
@@ -542,13 +542,13 @@ func (s *State) Start(ctx context.Context, schemaname string, migration *migrati
 		return nil, err
 	}
 
-	var schema schema.Schema
-	err = json.Unmarshal([]byte(rawSchema), &schema)
+	var unmarshalledSchema schema.Schema
```

Comment:
this change doesn't seem related to the pr?

Reply:
Just a small rename to stop a warning about shadowing a package name. I felt it was too small to include in a separate PR.